services/director/src/simcore_service_director/system_utils.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+from typing import Dict
+
+
+def get_system_extra_hosts(extra_host_domain: str) -> Dict:
+    extra_hosts = {}
+    hosts_path = Path("/etc/hosts")
+    if hosts_path.exists():        

```

Comment:
`` if hosts_path.exists() and extra_host_domain!="undefined": ``

Reply:
done